src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -693,6 +693,8 @@ namespace winrt::TerminalApp::implementation
 
         // Register for directory change notification.
         _RegisterSettingsChange();
+
+        jumplist.UpdateJumplist(_settings);

```

Comment:
Not sure if `AppLogic` is the right place to update the jumplist

Reply:
Doesn't this mean it's only updated once at startup?

Reply:
Oh it's updated on _ReloadSettings too... doesn't that get called once during startup already?

Reply:
Seems like `ReloadSettings` is called when updating my settings, and `LoadSettings` is called on startup, I could actually probably put this line inside of `TryLoadSettings` 🤔

Reply:
i'm fine with where it is (in both places) right now